packages/babel-preset-udemy-website/tests.js

```diff
@@ -11,8 +11,8 @@ console.log('test babel-preset-udemy-website');
 
 fs.readdirSync(join(__dirname, 'tests/source')).filter(name => name.endsWith('.js')).forEach(name => {
     const result = babel.transformFileSync(join(__dirname, 'tests/source', name), preset).code;
-    const expected = fs.readFileSync(join(__dirname, 'tests/result', name));
+    const expected = fs.readFileSync(join(__dirname, 'tests/result', name), { encoding: 'UTF-8' });
```

Comment:
These two changes make the tests a little bit easier to debug and a little more robust.